assets/javascripts/test_result.js

```diff
@@ -748,40 +748,61 @@ function renderInvestigationTab(response) {
 
     var tbodyElement = document.createElement('tbody');
     Object.keys(response).forEach(key => {
+        var value = response[key];
+        var type = 'pre';
+
+        //The value could be an object. This allows to extend the functionality
```

Comment:
I'd prefer a space after the `//`. I'd also replace could with can.

Is "This allows to extend the functionality this object will have the attribute type that determines the behavior" a complete sentence?

Reply:
Did both and rewrapped the paragraph and rewritten to be simpler and complete sentences.